zerver/decorator.py

```diff
@@ -172,7 +172,7 @@ def _wrapped_view_func(request, api_key=REQ,
                            *args, **kwargs):
 
         try:
-            user_profile = UserProfile.objects.get(api_key=api_key, is_active=True)
+            user_profile = UserProfile.objects.get(api_key=api_key, is_active=True, is_incoming_webhook=True)
         except UserProfile.DoesNotExist:
             raise JsonableError("Invalid API key")
```

Comment:
Some webhooks use `authenticated_rest_api_view` and above change will break it. It looks there is no reason to use `authenticated_rest_api_view` instead of `api_key_only_webhook_view`.

What do you think?